index.html

```diff
@@ -191,10 +240,16 @@ <h4 > LET’S CONNECT</h4>
                 <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
                 <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
             </ul>
+            <ul class="Programs-desclist">
+                <li   href="#"><img class="Programs-desck" src="./image/descktop/Frame 10 (1).png" alt=""></li>
+                <li  href="#"><img  class="Programs-desck" src="./image/descktop/Frame 11 (1).png" alt=""></li>
+                <li  href="#"><img  class="Programs-desck" src="./image/descktop/Frame 9 (2).png" alt=""></li>
+            </ul>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
         </ul>
     </div>
+</div>
 </section>
 <section class="cardcontact">
     <form class="cardcontact1" action="https://formspree.io/f/mwkzayzq" method="post" >
```

Comment:
Kindly make sure your design matches the provided Figma template. Right now there is some alignment issues in both the heading and the paragraph. ↓
Figma template:
<img width="564" alt="Screenshot 2022-12-16 at 10 13 17 PM" src="https://user-images.githubusercontent.com/25887253/208171862-82e2668e-a567-49cf-b8df-d52b6663c619.png">
Your design:
<img width="831" alt="Screenshot 2022-12-16 at 10 01 54 PM" src="https://user-images.githubusercontent.com/25887253/208171980-c804c6e9-3138-43fd-a4e3-b5d9dce5bfc0.png">